pages/Transactions.tsx

```diff
@@ -195,15 +195,16 @@ export function Transactions() {
           {[...Array(20)].map((e, i) => (
             <View
               key={i}
-              className="flex flex-row items-center text-sm gap-x-6 px-4 mb-4"
+              className="flex flex-row items-center gap-3 px-6 py-2 my-2"
             >
-              <Skeleton className="rounded-full w-10 h-10" />
-              <View className="flex flex-col flex-1 gap-1">
-                <Skeleton className="w-32 h-4" />
-                <Skeleton className="w-16 h-4" />
+              <Skeleton className="rounded-full w-12 h-12" />
+              <View className="flex flex-col flex-1 gap-2">
+                <Skeleton className="w-32 text-sm" />
```

Comment:
These are toned down a bit (than the font-size used for corresponding Texts) for aesthetic reasons (1 is current)

<img src="https://github.com/user-attachments/assets/6f745c2f-255e-4438-baf4-3d7d74e592d8" width="300" />
<img src="https://github.com/user-attachments/assets/9ab89b5a-12b0-44f3-901e-5a15ba4bfcf9" width="300" />